test/rpm/utils.sh

```diff
@@ -175,6 +229,14 @@ install_server() {
   # Make sure the RPMS are installed
   installed_rpms=$(rpm -q --qf "%{nvr}.%{arch} " go-fdo-server{,-{manufacturer,owner,rendezvous}})
   echo "  - Installed RPMs: ${installed_rpms}"
+
+  # Cleanup any stale test configuration
+  cleanup_drop_ins
+  cleanup_databases
+  cleanup_home_dirs
+
+  log_info "Creating server home directories"
+  setup_home_dirs
```

Comment:
I think it would make more sense to overwrite the `create_directories` function. Could you please copy the definition from `ci/utils.sh` to `rpms/utils.sh` and add a call to `setup_home_dirs` at the end?